web/src/components/storage/DriveEditor.tsx

```diff
@@ -393,7 +393,7 @@ const RemoveDriveOption = ({ drive }) => {
   return (
     <>
       <Divider component="hr" />
-      <MenuItem description={_("Remove the configuration for this device")}>
+      <MenuItem description={_("REMOVE the configuration for this device")} onClick={deleteDrive}>
```

Comment:
Why uppercase? 

For visually grabbing user attention about the destructive effect of the action  I'd use the isDanger MenuItem prop instead of playing with text that, somehow, is out of our control

Reply:
Sorry for confusing you, that is just my WIP debugging way to make sure my browser is seeing the intended branch